### PR TITLE
Save filled data on checkout and fill checkout with authorized user's data

### PR DESCRIPTION
**Pull request by dmytro07**
Inputs field on checkout form are field by user's data if user is authorized
Screenshot from 2021-04-08 17-47-01

Also all field data now are saved when you moved to another page and back to the checkout

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by yuliiakucher — DISMISSED** [review]

File: src/utils/checkout.js [hunk 1]
```diff
@@ -189,3 +190,58 @@ export const POST_OFFICE_NUMBER = 'Відділення № ';
...
+      city,
+      street,
+      house,
+      flat
+    };
+  }
+  return result;
+};
+
+const COURIER = 'COURIER';
```
Comment (twihell) [inline src/utils/checkout.js:214]: Is it ok if we have magic strings within util files? Even if it's not regulated strictly, there is still **consts/** folder, so maybe it's better to move this const there.

File: src/utils/checkout.js [hunk 2]
```diff
@@ -189,3 +190,58 @@ export const POST_OFFICE_NUMBER = 'Відділення № ';
...
+      house: '',
+      region: '',
+      regionId: '',
+      street: '',
+      userComment: ''
+    });
+  }
+};
+
+export const RESET = 'reset';
```
Comment (twihell) [inline src/utils/checkout.js:245]: Same here

File: src/configs/index.js [hunk 3]
```diff
@@ -454,3 +454,10 @@ export const deliveryTypes = {
 };
 
 export const CY_CODE_ERR = 'code-error';
+
+export const SESSION_STORAGE = {
+  CHECKOUT_FORM: 'checkoutForm',
+  DELIVERY_TYPE: 'deliveryType'
+};
+
+export const HORONDI = 'horondi';
```
Comment (twihell) [inline src/configs/index.js:463]: you can use this const in  **src/index.js**  in localStorage/sesstionStorage conditionals

**Review by grebiniuk — CHANGES_REQUESTED** [review]
Try to fix warehouses for NovaPoshta on checkout.

**Review by twihell — APPROVED** [review]

**Review by grebiniuk — APPROVED** [review]
Well done!

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**